src/vignette/storage/static_assets.clj

```diff
@@ -31,10 +51,23 @@
           (if (= 451 status)
             (first-available (rest image-urls))))))))
 
-(defrecord StaticImageStorage [static-image-url] ImageStorageProtocol
-  (save-thumbnail [_ _ _] nil)
-  (get-thumbnail [_ _] nil)
-  (save-original [_ _ _] nil)
+(defrecord StaticImageStorage [store, static-image-url] ImageStorageProtocol
+  (save-thumbnail [this resource thumb-map]
+      (put* (:store this)
+        resource
+        thumb-map
+        mt/thumbnail-path))
+
+  (get-thumbnail [this thumb-map]
+       (get* (:store this)
+         thumb-map
+         mt/thumbnail-path))
```

Comment:
`thumbnail-path` creates a path from keys such as `top-dir` and  `middle-dir`. Do we have these in `thumb-map` for static assets?